src/node.rs

```diff
@@ -1409,7 +1410,11 @@ mod tests {
             node.msg_log.add_message(message, state).unwrap();
         }
 
-        block.payload = node.build_seal(state).unwrap();
+        let seal = node.build_seal(state).unwrap();
+        block.payload = seal
+            .write_to_bytes()
+            .map_err(|err| PbftError::SerializationError("Error writing seal to bytes".into(), err))
```

Comment:
Probably can just implement `From` for this error, given the number of times you've done this.

Reply:
Okay if I include implementing `From` for error types in a different PR, or would you prefer I do that in this one?